src/daemon/daemon_rpc.cpp

```diff
@@ -32,28 +32,24 @@ namespace
 {
 constexpr auto category = "rpc";
 
-void throw_if_server_exists(const std::string& address)
+bool see_if_im_in_the_kitchen(const std::string& address)
```

Comment:
Although I think the name of this function is funny, I think we should stick to something along the lines of `check_if_server_running()` or something.

Reply:
Of course, I will fix that  :sweat_smile: